.circleci/release_s3.sh

```diff
@@ -4,8 +4,34 @@ set -eu
 
 VERSION=$(cat VERSION)
 
+# Synchronises a folder with the aws and gcp buckets.
```

Comment:
Thanks ChatGPT! I not longer have to anguish over the bash syntax, and can just ask you to generate this for me :D